src/app.js

```diff
@@ -1,5 +1,199 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
This shouldnt really be needed. The rules are there for a reason, this is getting flagged up because you are using ternary operators to determine what to return in several of the functions. This is easily fixed by the use of an if block (Ill add suggestions to the affected functions for you), so you should delete this line